geometry/segment2d.cpp

```diff
@@ -2,10 +2,16 @@
 
 #include "geometry/robust_orientation.hpp"
 
-#include "std/algorithm.hpp"
+#include <algorithm>
```

Comment:
Empty line between `include` and `using namespace` please

Reply:
Done